lib/util.js

```diff
@@ -38,18 +69,12 @@ function encodeClaim (amount, id) {
   return Buffer.concat([
     Buffer.from('CLM\0'),
     Buffer.from(id, 'hex'),
-    bignum(amount).toBuffer({
-      endian: 'big',
-      size: 8
-    })
+    toU64BE(amount)
```

Comment:
Uint64 exceeds the max amount a claim can have. See https://github.com/ripple/ripple-lib/blob/develop/docs/index.md#value

Reply:
Valid amounts should pass this check

```js
/^([1-9][0-9]{0,16})$|^100000000000000000$/.test(amount)
```